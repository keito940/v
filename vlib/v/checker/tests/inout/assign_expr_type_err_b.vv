fn main() {
	mut a := true
	a *= false
}