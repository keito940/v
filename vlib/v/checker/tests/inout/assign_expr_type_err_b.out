vlib/v/checker/tests/inout/assign_expr_type_err_b.v:3:7: error: operator *= not defined on left type `bool`
    1| fn main() {
    2|     mut a := true
    3|     a *= false
                ~~~~~
    4| }
